### PR TITLE
Merge create schema sql and zkapp tables sql

### PR DESCRIPTION
**Pull request by dkijania**
This is my proposal to merge `create_schema.sql` and `zkapp_tables.sql`  since it is easier to just run psql with single script not many. In second case we need to make sure our pwd is in src/app/archive folder which has bad usability for scripting or manual archive handling. I also found that we don't ship create_schema script in archive debian

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: scripts/deb-builder-helpers.sh [hunk 1]
```diff
@@ -349,7 +349,8 @@ build_archive_deb () {
   cp ./default/src/app/replayer/replayer.exe "${BUILDDIR}/usr/local/bin/mina-replayer"
   cp ./default/src/app/swap_bad_balances/swap_bad_balances.exe "${BUILDDIR}/usr/local/bin/mina-swap-bad-balances"
 
-
+  cp ../src/app/archive/*.sql "${BUILDDIR}/etc/mina/archive"
```
Comment (georgeee) [inline scripts/deb-builder-helpers.sh:352]:
```suggestion
  cp ../src/app/archive/create_schema.sql "${BUILDDIR}/etc/mina/archive"
```

**Review by georgeee — APPROVED** [review]

**Review by rbonichon — APPROVED** [review]
I think it's good change. Would have helped me find some information faster ;-)

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-nightly-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Pull request merged**